frontend/src/Store/features/auth/authSlice.ts

```diff
@@ -34,42 +49,96 @@ function buildInitialState(): AuthState {
     };
 }
 
+const handleAuthResponse = createAsyncThunk<
+    any,
+    Promise<AuthResponse>,
+    { dispatch: AppDispatch }
+>('auth/handleAuthResponse',
```

Comment:
Pewnie korzystasz z prettiera? Ja bym dał tutaj `//perttier-ignore` żeby nie formatowało tego. Mnie się trudniej czyta w takiej postaci. Ale to tylko luźna sugestia

Reply:
Nie prettier, ale nie podobało mi się w 1 linii. Tak jak jest też mi się nie podoba to wrzucę do 1 żeby chociaż komuś się podobało